### ISSUE TITLE
Unexpected errors linting javascipt - no apparent pattern

### ISSUE THREAD

**Issue by stuartish** [revelrylabs/lintron#212]
## Background

Unable to identify a pattern with these errors.

Error on a line that starts a comment: https://github.com/revelrylabs/TSS/pull/5076#discussion_r269367175

Assigning a constant:
https://github.com/revelrylabs/TSS/pull/4958/files/c15e181cd7a9bdc80aa178050f884f55bf66a8a7#r267109510

Some logic and opening a block:
https://github.com/revelrylabs/TSS/pull/5139#discussion_r270429505

Just another line in some json? 
https://github.com/revelrylabs/TSS/pull/5139#discussion_r270429512

Declaring a function:
https://github.com/revelrylabs/TSS/pull/4500#discussion_r252784108

**Comment by stuartish** [comment]
Sought to get some more info about what's up with these by adding logging of the actual error message to output. From es_lint.rb:
```
      rescue JSON::ParserError => msg
        [
          Violation.new(
            file: file,
            line: (file.patch.changed_lines.first.number rescue 1),
            message: 'Unexpected error in ESLint:' << msg,
```

It seems I can't push code to this repo. Will see about testing this locally.

**Comment by stuartish** [comment]
Tried to figure out how to run this locally but not going well. I need to learn more about linter configs also.

**Comment by grossvogel** [comment]
@stuartish I think this should be fixed now. Can you verify and close if that's true?

**Comment by prehnRA** [comment]
I saw this work on a project this morning, so we're good to close.
